pkg/sql/privilege/privilege.go

```diff
@@ -236,3 +236,47 @@ func GetValidPrivilegesForObject(objectType ObjectType) List {
 		panic(errors.AssertionFailedf("unknown object type %s", objectType))
 	}
 }
+
+// PrivilegeListToACL converts a list of privileges to a list of Postgres
+// ACL items.
+// See: https://www.postgresql.org/docs/13/ddl-priv.html#PRIVILEGE-ABBREVS-TABLE
+//     for privileges and their ACL abbreviations.
+func PrivilegeListToACL(privileges List, objectType ObjectType) string {
+	// If ALL is present, explode ALL into the underlying privileges.
+	if privileges.Contains(ALL) {
+		privileges = GetValidPrivilegesForObject(objectType)
+	}
+	chars := make([]string, len(privileges))
+	for _, privilege := range privileges {
+		switch privilege {
+		case CREATE:
+			chars = append(chars, "C")
+		case DROP:
+			// DROP is not a Postgres privilege.
```

Comment:
hm i think it may be safer to not include the ones that Postgres doesn't support in the output. the table is for compatibility, so i don't think there will really be any tools looking for DROP/GRANT/ZONECONFIG in this table.

Reply:
I opted to include it since I'm planning on re-using this table to implement `SHOW DEFAULT PRIVILEGES` but I think it would be reasonable to keep this as postgres compatible as possible and create a new crdb_internal table to use for showing the output, how does that sound?

Reply:
hmmm interesting. yeah crdb_internal sounds safer, though kind of unfortunate that it will duplicate some logic. i wouldn't be surprised if a later Postgres release starts using these letters, then we'd be in trouble

Reply:
Removed for now.